### DISCUSSION TITLE
can you make example how to use jwt from response type id token?

### DISCUSSION THREAD

**Discussion by jdnielss** [adrianbrs/nest-oidc-provider#18] [Q&A] [answered]
I want to use jwt to validate from other services, but I don't know how to use it... i use response type id_token

can you make an example of how to use and consume it?

thank you in advance

**Comment by adrianbrs** [comment, marked as answer]
This lib is just a wrapper to help integrate `oidc-provider` with NestJS, so this type of question is best suited for the oidc-provider package and should be asked in the correct repository, so that you can get better support.

Please take a look at the [oidc-provider documentation](https://github.com/panva/node-oidc-provider/blob/main/docs/README.md) or open a [discussion in their repository](https://github.com/panva/node-oidc-provider/discussions) if you need.

Thanks!
